corehq/apps/enterprise/utils.py

```diff
@@ -0,0 +1,19 @@
+from corehq.apps.enterprise.models import EnterprisePermissions
+
+
+def get_enterprise_domains(domain):
+    """Return list containing input domain and source domain for that input domain."""
+    domain_list = [domain]
+    config = EnterprisePermissions.get_by_domain(domain)
+    if config.is_enabled and domain in config.domains:
+        domain_list.append(config.source_domain)
```

Comment:
This will return `[domain, domain]` if the given domain is itself a source domain, won't it? I think that's a little counterintuitive but not harmful.

Reply:
If the domain is the source domain it won't pass the condition `domain in config.domains` (`config.domains` doesn't include the source). So if `domain` is a source domain the method should just return `[domain]`.

Reply:
+1

Reply:
thanks 💡